src/Raven.Server/Rachis/Candidate.cs

```diff
@@ -94,17 +94,7 @@ private void Run()
                     {
                         if (_peersWaiting.WaitOne(_engine.Timeout.TimeoutPeriod) == false)
                         {
-                            ElectionTerm = _engine.CurrentTerm;
-
-                            // timeout? 
-                            if (IsForcedElection)
-                            {
-                                CastVoteForSelf(ElectionTerm + 1, "Timeout during forced elections", setStateChange: false);
```

Comment:
At the start of the candidate we already check if this is forced and casting the vote.

Reply:
This can happen on the _next_ term, though.

Reply:
Not sure I'm following.. In force election we casting the vote and increment the term to give to this node an advantage for the coming elections. (it might happened that the other nodes are already during an election process and have higher terms..)